workers/thermostat.py

```diff
@@ -26,10 +27,10 @@ def __init__(self):
       self._mapped_modes = {
         Mode.Closed: STATE_OFF,
         Mode.Open: STATE_ON,
-        Mode.Auto: STATE_HEAT,
+        Mode.Auto: STATE_AUTO,
```

Comment:
Is this correct with newest HA ?

Reply:
This is working in Home Assistant 0.94, yes